module/VuFind/src/VuFind/View/Helper/Root/Record.php

```diff
@@ -413,7 +413,8 @@ public function getCheckbox($idPrefix = '', $formAttr = false, $number = null)
         $id = $this->driver->getSourceIdentifier() . '|'
             . $this->driver->getUniqueId();
         $context
-            = ['id' => $id, 'number' => $number, 'prefix' => $idPrefix];
+            = ['id' => $id, 'number' => $number, 'prefix' => $idPrefix,
+               'title' => $this->getTitleHtml()];
```

Comment:
I would recommend calling this value `titleHtml` instead of just `title`. This makes it more clear that it does not need additional escaping applied in the checkbox.phtml template -- otherwise I think there's a chance that in future reviews this might trigger people to reinvestigate code unnecessarily.